tao_compiler/mlir/xla/ral/context/common_context_impl_pdll.cc

```diff
@@ -55,6 +55,36 @@ MemRefType<T, N> simple_test_fused_add_mul_kernel(ExecutionContext* ctx,
   // Note that `0.001` is safe to check exact match even for float type.
   TAO_CHECK(float0_001 == 0.001);
 
+  auto& intArrayAttr =
+      dictAttr.get("array5int0int1int2").template as<ArrayPDLAttr>();
+  TAO_CHECK(intArrayAttr.size() == 3);
+  TAO_CHECK(intArrayAttr.get(1).template as<IntPDLAttr>().getValue() == 1);
```

Comment:
please check all the elements, not just one.

Reply:
fixed